docs/getting-started/supported-languages-and-tools.md

```diff
@@ -212,7 +228,11 @@ The table below lists all languages that Codacy supports and the corresponding t
       <td>Helm</td>
       <td>-</td>
       <td>-</td>
-      <td><a href="https://trivy.dev">Trivy</a> <a href="#yaml-only"><sup>2</sup></a></td>
+      <td>
+          <!--
```

Comment:
to uncomment this we need to first add Language.YAML as a supported language to Semgrep in codacy/codacy-tools